eq-author/src/components/MainCanvas/index.js

```diff
@@ -1,10 +1,14 @@
 import styled from "styled-components";
 
 const MainCanvas = styled.div`
-  max-width: 55em;
+  width: ${props => props.width};
   margin: 0 auto 1em;
   padding: 0 1em;
   position: relative;
 `;
 
+MainCanvas.defaultProps = {
+  "min-width": "55em",
```

Comment:
Default prop is `min-width`? But the read prop is `width`?

Reply:
This is likely copied from the prototype, probably just needs removing.

Reply:
Yes, thats correct. I changed it to width.